quarkchain/cluster/cluster_config.py

```diff
@@ -138,6 +138,8 @@ class ClusterConfig(BaseConfig):
     PRIVATE_JSON_RPC_PORT = 38491
     JSON_RPC_HOST = "localhost"
     PRIVATE_JSON_RPC_HOST = "localhost"
+    ENABLE_PUBLIC_JSON_RPC = False
```

Comment:
we may want to be backward-compatible to avoid surprises. consider defaulting to `True`?

Reply:
Make sense.